raiden/transfer/node.py

```diff
@@ -794,6 +803,11 @@ def handle_state_change(chain_state: ChainState, state_change: StateChange) -> T
             chain_state,
             state_change,
         )
+    elif type(state_change) == ActionUpdateTransportSyncToken:
```

Comment:
It's probably a bit of an overkill but perhaps also a unit test for this?

Reply:
I swear i had it somewhere... must have lost the change when i got myself into rebase conflicts :man_facepalming:

Reply:
Done